### ISSUE TITLE
[Issue] page for edit unit group not found

### ISSUE THREAD

**Issue by ivalrivall** [Blair2004/NexoPOS#1986]
**Describe the bug**
page not found

**To Reproduce**
1. Create unit groups
2. Click on options
3. Choose edit

**Expected behavior**
Page edit is showing

**Environment:**
 - NexoPOS vesrion: 5.2.5

**Comment by nzian** [comment]
Yes its a bug. In the Crud file app\Crud\UnitGroupCrud.php url paths was written wrong. delete also not work. I am going to create a pull request now

**Comment by Blair2004** [comment]
This should be fixed on v5.2.6
